src/main/java/com/eternalcode/discordapp/data/DataManager.java

```diff
@@ -0,0 +1,36 @@
+package com.eternalcode.discordapp.data;
+
+import com.eternalcode.discordapp.config.CdnConfig;
+import net.dzikoysk.cdn.Cdn;
+import net.dzikoysk.cdn.CdnFactory;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.Set;
+
+public class DataManager {
```

Comment:
ConfigurationLoader prędzej. Albo Yml data loader

Reply:
To jest klasa wykorzystywana wyłącznie do plików z danymi generowanymi przez bota - postaram się przerobić ConfigManager by był uniwersalny zamisat stosować dwóch tych samych klas